locale/eo.js

```diff
@@ -36,25 +36,25 @@
         calendar : {
             sameDay : '[Hodiaŭ je] LT',
             nextDay : '[Morgaŭ je] LT',
-            nextWeek : 'dddd [je] LT',
+            nextWeek : 'dddd[n je] LT',
             lastDay : '[Hieraŭ je] LT',
-            lastWeek : '[pasinta] dddd [je] LT',
+            lastWeek : '[pasintan] dddd[n je] LT',
             sameElse : 'L'
         },
         relativeTime : {
             future : 'post %s',
             past : 'antaŭ %s',
-            s : 'sekundoj',
+            s : 'kelkaj sekundoj',
```

Comment:
Adding the word "kelkaj" (=some), because "post sekundoj" and "antaŭ sekundoj" don't really make sense.